src/utils/languageLocalization/validationText.ts

```diff
@@ -25,36 +17,33 @@ export const validationText: LanguageLocalizationType = {
     en: 'Required captcha',
     ru: 'Символы с картинки обязательны для ввода',
   },
-  minName: {
-    en: 'Must be not less than 3 symbols',
-    ru: 'Должно быть не менее 3 символов',
-  },
-  maxName: {
-    en: 'Must be not more than 30 symbols',
-    ru: 'Должно быть не более 30 символов',
-  },
   requiredName: {
     en: 'Required full name',
     ru: 'Полное имя обязательно для ввода',
   },
-  maxPublication: {
-    en: 'Must be not more than 100 symbols',
-    ru: 'Должно быть не более 100 символов',
-  },
   requiredPublication: {
     en: 'Required publication',
     ru: 'Публикация обязательна для ввода',
   },
-  maxChatText: {
-    en: 'Must be not more than 200 symbols',
-    ru: 'Должно быть не более 200 символов',
-  },
   requiredChatText: {
     en: 'Required text',
     ru: 'Текст обязателен для ввода',
   },
-  maxTerm: {
-    en: 'Must be not more than 50 symbols',
-    ru: 'Должно быть не более 50 символов',
+
+  min3: {
+    en: 'Must be not less than 3 symbols',
+    ru: 'Не менее 3 символов',
+  },
+  max30: {
+    en: 'Must be not more than 30 symbols',
+    ru: 'Не более 30 символов',
+  },
+  max100: {
+    en: 'Must be not more than 100 symbols',
+    ru: 'Не более 100 символов',
+  },
+  max200: {
+    en: 'Must be not more than 200 symbols',
+    ru: 'Не более 200 символов',
   },
```

Comment:
I would propose to make it more configurable =)
I could propose to create new util like
```
export const stringReplacer = (string, params, flag = 'i') => {
  let result = string;
  for (const [key, value] of Object.entries(params)) {
    const regexp = new RegExp(`${key}`, flag);
    result = result.replace(regexp, value);
  }
  return result;
};
```
And use it like:
```
 maxTerm: {
    en: 'Must be not more than %{number} symbols',
    ru: 'Должно быть не более %{number} символов',
    }

stringReplacer(validationText.maxTerm[languageMode], { '%{number}': 30})
stringReplacer(validationText.maxTerm[languageMode], { '%{number}': 50})
```
As result you will have only one string with dynamic param

Reply:
Done!